src/app/containers/AVPlayer/index.jsx

```diff
@@ -8,6 +14,15 @@ import pathOr from 'ramda/src/pathOr';
 import { RequestContext } from '#contexts/RequestContext';
 import { ServiceContext } from '#contexts/ServiceContext';
 
+const Wrapper = styled.div`
```

Comment:
Could we call this `StyledAVPlayer` instead of `Wrapper`?

Reply:
So at the OnDemandTvPage level (above the wrapper) there is already a `StyledVideoPlayer` (which is a styled version of AVPlayer), and in the psammead-media-player (level below) there's a `StyledVideoContainer`... I dunno I just find all the repetition of `Styled` & repeated 'player/video' declarations a bit confusing, and in this instance - it isn't the AV player. It's a div that wraps the AV player.

We don't seem to have a consistent approach to this though, so 🤷‍♂️

Reply:
Fair enough, let's leave it as is